test/e2e/tests/import-flow.spec.js

```diff
@@ -22,6 +22,8 @@ const ganacheOptions = {
   ],
 };
 
+const SMALL_DELAY = 2000;
```

Comment:
Take a look if really required

Reply:
Removed in cd22f628dd2554cfb019dcfa495e12a5ebe0eb4b